scalafix-tests/integration/src/test/scala/scalafix/tests/reflect/RuleDecoderSuite.scala

```diff
@@ -34,7 +34,7 @@ class RuleDecoderSuite extends AnyFunSuite {
     assert(expectedName == rules.name.value)
   }
 
-  test("relative resolves from custom working directory") {
+  test("relative resolves from custom working directory", SkipWindows) {
```

Comment:
`integration3` fails on windows because of this
```
[info] - relative resolves from custom working directory *** FAILED *** (34 milliseconds)
[info]   java.nio.file.InvalidPathException: Illegal char <:> at index 2: /D:/a/scalafix/scalafix/scalafix-tests/integration/target/jvm-3/test-classes/
[info]   at java.base/sun.nio.fs.WindowsPathParser.normalize(WindowsPathParser.java:182)
[info]   at java.base/sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:153)
[info]   at java.base/sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:77)
[info]   at java.base/sun.nio.fs.WindowsPath.parse(WindowsPath.java:92)
[info]   at java.base/sun.nio.fs.WindowsFileSystem.getPath(WindowsFileSystem.java:232)
[info]   at java.base/java.nio.file.Path.of(Path.java:147)
[info]   at java.base/java.nio.file.Paths.get(Paths.java:69)
[info]   at dotty.tools.dotc.core.MacroClassLoader$.$anonfun$1(MacroClassLoader.scala:25)
[info]   at scala.collection.immutable.List.map(List.scala:246)
[info]   at dotty.tools.dotc.core.MacroClassLoader$.makeMacroClassLoader(MacroClassLoader.scala:25)
```